docs/development/extensions-core/bloom-filter.md

```diff
@@ -23,28 +23,25 @@ title: "Bloom Filter"
   -->
 
 
-To use this Apache Druid extension, [include](../../configuration/extensions.md#loading-extensions) `druid-bloom-filter` in the extensions load list.
+To use the Apache Druid&circledR; Bloom filter extension, include `druid-bloom-filter` in the extensions load list. See [Loading extensions](../../configuration/extensions.md#loading-extensions) for more information.
 
-This extension adds the ability to both construct bloom filters from query results, and filter query results by testing
-against a bloom filter. A Bloom filter is a probabilistic data structure for performing a set membership check. A bloom
-filter is a good candidate to use with Druid for cases where an explicit filter is impossible, e.g. filtering a query
+This extension adds the ability to both construct Bloom filters from query results, and filter query results by testing
+against a Bloom filter. A Bloom filter is a probabilistic data structure for performing a set membership check. A Bloom
+filter is a good candidate to use with Druid for cases where an explicit filter is impossible, such as filtering a query
```

Comment:
```suggestion
filter is a good candidate to use when an explicit filter is impossible, such as filtering a query
```
Don't think we need "for Druid" since these are the Druid docs